### PR TITLE
Add `worker_main_http_uri` to the contrib bash script

### PR DESCRIPTION
**Pull request by villepeh**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

This small code change adds _worker_main_http_uri: http://localhost:8008/_ to the example bash script. Also changed >> to > to overwrite previous files rather than appending into them

Signed-off-by: Ville Petteri Huh.

### CODE REVIEW

**Comment by villepeh** [comment]
The variable is part of the example generic_worker too: https://github.com/matrix-org/synapse/blob/develop/docs/systemd-with-workers/workers/generic_worker.yaml

File: changelog.d/13772.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Add worker_main_http_uri for the worker generator bash script.
```
Comment (dklimpel) [inline changelog.d/13772.doc:1]:
```suggestion
Add `worker_main_http_uri` for the worker generator bash script.
```

**Review by erikjohnston — APPROVED** [review]
Makes sense, thanks!

**Comment by DMRobertson** [comment]
I'm going to manually merge this and assume the complement failures are yet more flakes.

**Pull request merged**